app/api/profile.ts

```diff
@@ -49,6 +51,7 @@ class ProfileAPI extends PlutoAxios {
       id: params.id,
       title: params.title,
       received_date: params.receivedDate,
+      related_link: !params.relatedLink ? null : params.relatedLink,
```

Comment:
When you use conditional ternary operator, please use a positive expression as much as possible for the condition.
description are below.